### PR TITLE
Add filter for ads.txt content.

### PR DESCRIPTION
**Pull request by ethanclevenger91**
### Description of the Change

Enables filtering of the output ads.txt content.

### Benefits

See #35 

### Verification Process

I don't see any formal tests, but I was able, in a test environment, to utilize the filter:

```
add_filter('ads_txt_content', function($content) {
  return $content . PHP_EOL . 'foo';
});
```

Results were as expected - my existing ads.txt configured content was output with `foo` on a new line at the end.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [x] All new and existing tests passed.

### Applicable Issues

#35

### CODE REVIEW

File: ads-txt.php [hunk 1]
```diff
@@ -31,7 +31,16 @@ function tenup_display_ads_txt() {
...
+			$adstxt = esc_html( $post->post_content );
+
+			/**
+			 * Filter the ads.txt content.
+			 *
+			 * @since x.x.x
+			 *
+			 * @param type  $adstxt The existing ads.txt content.
+			 */
+			echo apply_filters( 'ads_txt_content', $adstxt );
```
Comment (adamsilverstein) [inline ads-txt.php]: can we apply the `esc_html` here at the point of output (and remove it above). For security reasons, we always want to **escape on output**.

File: ads-txt.php [hunk 2]
```diff
@@ -31,7 +31,16 @@ function tenup_display_ads_txt() {
 		if ( ! empty( $post_id ) ) {
 			$post = get_post( $post_id );
 			header( 'Content-Type: text/plain' );
-			echo esc_html( $post->post_content );
+			$adstxt = esc_html( $post->post_content );
+
+			/**
+			 * Filter the ads.txt content.
+			 *
+			 * @since x.x.x
```
Comment (adamsilverstein) [inline ads-txt.php]: This seems good for 1.2.0
```suggestion
			 * @since 1.2.0
```

**Comment by adamsilverstein** [comment]
Hey @ethanclevenger91 this looks great, thanks for contributing! I left some small change requests, mainly moving the escaping to the output line. Let me know if you have any questions.

**Comment by ethanclevenger91** [comment]
@adamsilverstein you got it!

**Pull request merged**